addon/ng2/blueprints/ng2/files/typings.json

```diff
@@ -6,5 +6,6 @@
     "jasmine": "github:DefinitelyTyped/DefinitelyTyped/jasmine/jasmine.d.ts#26c98c8a9530c44f8c801ccc3b2057e2101187ee"
   },
   "ambientDependencies": {
+    "es6-shim": "github:DefinitelyTyped/DefinitelyTyped/es6-shim/es6-shim.d.ts#6697d6f7dadbf5773cb40ecda35a76027e0783b2"
```

Comment:
Since angular needs only es6 collections and es6 promises, do we want pull in the whole es6-shim typings? While  convenient, this is more than what is necessary.

This is all that angular's public api surface requires: https://github.com/angular/angular/tree/builds-js/typings